cmd/geth/consolecmd.go

```diff
@@ -112,7 +113,11 @@ func localConsole(ctx *cli.Context) error {
 // console to it.
 func remoteConsole(ctx *cli.Context) error {
 	// Attach to a remotely running geth instance and start the JavaScript console
-	client, err := dialRPC(ctx.Args().First())
+	endpoint := ctx.Args().First()
+	if (endpoint == "") && (ctx.GlobalIsSet(utils.DataDirFlag.Name)) {
```

Comment:
Please remove unnecessary parens here. You can write `endpoint == "" && ctx.GlobalIsSet(utils.DataDirFlag.Name)`.

Reply:
@fjl sure, fixed